lib/simctl.ts

```diff
@@ -120,11 +121,25 @@ export class Simctl implements ISimctl {
 		return devices;
 	}
 
+	public getLog(deviceId: string, predicate?: string): any {
+		let predicateArgs: string[] = [];
+
+		if (!_.isUndefined(predicate)) {
```

Comment:
I think a simple check like `if (predicate) {` would suffice here, but this is a personal preference